service/pom.xml

```diff
@@ -189,7 +189,19 @@
             </plugin>
         </plugins>
     </build>
-    <!--<repositories>-->
+    <repositories>
```

Comment:
this is added in certprocessor pom.xml , so it is required here as well.

Reply:
As